.github/workflows/schema-ci.yml

```diff
@@ -0,0 +1,23 @@
+name: schema-ci
+
+on:
+  pull_request:
+    paths:
+      - "schemas/**"
+    branches: [main]
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  schema-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
I think we can try `actions/checkout@v4`

Reply:
ok

Reply:
done and tested.
It works.